src/styles/app/05-main-area.css

```
@@ -48,6 +48,25 @@
 	background: var(--bg-primary);
 }
 
+@keyframes homeContentZoomIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "homeContentZoomIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)